src/calendar/day/custom/index.js

```diff
@@ -1,10 +1,22 @@
 import React, {Component} from 'react';
-import {TouchableOpacity, Text} from 'react-native';
+import {View, TouchableOpacity, Text} from 'react-native';
 import PropTypes from 'prop-types';
 
 import styleConstructor from './style';
 import {shouldUpdate} from '../../../component-updater';
 
+const styleVisualAccesibility = {borderColor: '#1B0088', borderWidth: 1, position: 'absolute', top: -2};
+
+const VisualAccessibility = ({state, children}) => {
```

Comment:
Why do we need this?
We can override the style easily using:
```
'stylesheet.day.single': {
    today: {borderColor: '#1B0088', borderWidth: 0.5}
  }
```
Am I missing something?
Please remove VisualAccessibility

Reply:
I need this, could it be optional?, with a external props todayVisualAccessibility with default false?, 

Is a only border in the day "today" .